docs/gradle-plugin/migrations/migration-to-0.7.0.md

```diff
@@ -618,28 +609,28 @@ See [migrate instruction](#merge-reports-config-was-removed).
 ### Using 'IntellijEngine' is an error
 _Solution_
 
-Use function `useKoverTool()` instead.
+Remove the engine change, the Kover toolkit is used by default
```

Comment:
Are you sure that there are no use-cases when one have to programmatically revert to using Kover tool?

Reply:
M.b. add function `useJacoco(Boolean)`?

Reply:
No, boolean parameter can be confusing. I think we can leave this as is until users come and tell about use-cases